src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+text.
+
+### new
+
+New requires gender for many languages. Add is a better substitute. Features
+that are new to the user are best communicated through emails, landing pages,
+and marketing materials.
+
+### OK
+
+Avoid OK as a button label. Button labels should make it clear what’s going to
+happen when the user clicks it.
+
+On mobile devices it’s permitted to use OK for button text.
+
+### OAuth
+
+Not oAuth.
+
+### on, onto
+
+You record conversations on a ticket. Onto conveys actual motion. For example, I
+jumped onto the parallel bars. If you’re confused, just say up before on in the
+sentence. If it makes sense, onto is the right word.
+
+### plan, account
+
+Don’t use account to refer to Zendesk plans.
+
+<BestPractice>
+  <Do>
+    <p>The Enterprise plan supports light agents.</p>
+  </Do>
+  <Dont>
+    <p>The Enterprise account supports light agents.</p>
+  </Dont>
+</BestPractice>
+
+### plugin
+
+No hyphen.
+
+### product tray
+
+Use products icon instead.
+
+### register vs. sign up
+
+Use register instead.
+
+### self-service
+
+Always hyphenate.
+
+### set up, setup
+
+Two words (set up) for the verb, one word (setup) for the adjective or as a
+noun.
+
+### sign in, sign out
+
+Use sign in and sign out instead of log in, and log out. Other forms: signed in,
+signed out. Hyphenate as an adjective or as a noun.
+
+<BestPractice>
+  <Do>
+    <p>Sign in</p>
+  </Do>
+  <Dont>
+    <p>Log in</p>
+  </Dont>
+</BestPractice>
+
+### single sign-on (SSO)
+
+Hyphenate sign-on and spell this out on first use.
+
+### spam
+
+Don’t use spam as a verb.
+
+### subdomain
+
+One word.
+
+### subscription
+
+For a person’s Zendesk account only. Not for individual plans or products.
+
+### support, Support
+
+Capitalize when referring to the Zendesk Support product. Use lowercase
+otherwise.
+
+### Support team
+
+Don’t use. Use Zendesk Customer Support or Customer Support instead.
+
+### stats
+
+Used in Help Center articles and in reports. Abbreviation OK.
+
+### thank you, thanks
+
+Reserve thank you and thanks when users provide information that’s difficult or
+inconvenient to collect.
+
+<BestPractice>
+  <Do>
+    <p>Thank you for submitting a request.</p>
+  </Do>
+  <Dont>
+    <p>Thank you for changing your settings.</p>
+  </Dont>
+</BestPractice>
+
+### ticket, ticket form
+
+Lowercase for both. Don’t capitalize for emphasis.
+
+### trigger
+
+Lowercase.
+
+### user name
+
+Two words. Not hyphenated.
+
+### website, web form, web page
+
+All lowercase.
+
+### Web Widget
+
+Capitalize both words.
+
+### workflow
+
+One word.
+
+### Zendesk
+
+Always capitalize the name of our company. Never use ZenDesk or give it an
+apostrophe. We’re not the possessive type.
+
+### Zendesk Apps
+
+Capitalize this feature name: “The Zendesk Apps framework.”
+
+<hr />
+
+## Words we avoid
+
+These words aren’t bad. They’re just not in our terminology diet.
+
+### oops, whoops, oh no
+
+Oops, we did it again. Seriously, though, we don’t use oops or other
```

Comment:
Oops, we did it again. Seriously, though, we don’t use _oops_ or other self-deprecating blurts in error messages or empty states.